src/frontend/app/features/metrics/metrics/metrics.component.ts

```diff
@@ -40,6 +90,26 @@ export class MetricsComponent {
       ])),
       first()
     );
-  }
 
+    const metricsMonitor = this.entityMonitorFactory.create<IMetrics>(
+      metricsAction.metricId,
+      metricSchemaKey,
+      entityFactory(metricSchemaKey)
+    );
+
+    this.jobDetails$ = metricsMonitor.entity$.pipe(
+      filter(a => !!a),
+      map((targetsData: any) => {
+        const mapped = {};
+        if (targetsData.activeTargets) {
```

Comment:
This is `targetsData.data.activeTargets` in latest metrics branch, not sure if it's an issue yet on v2-master